en/docs/get-started/dynamic-client-registration.md

```diff
@@ -14,6 +14,25 @@ This page explains how to onboard TPP applications using the Dynamic Client Regi
 
     3. Restart the Identity Server.
 
+    4. If you are using **WSO2 API Manager 4.2.0**, change the API Manager REST API version.
```

Comment:
What about the previous versions? Dont we need to add what versions need to be used?

Reply:
I guess we will have to include something similar to the change made in configuring-api-manager-for-ob.md page